frontend/src/actions/submitNewLoan.js

```diff
@@ -0,0 +1,40 @@
+// submitNewLoan
+// Basically add to reducers
+// backNewLoan
+// Basically mapStateToProps all saved info
+
+export const submitNewLoan = (
```

Comment:
Hmm I think Alan already added Redux for this component we can talk about it more in person

Reply:
Follow up, I think the main question for the both of you is about the redux in this code.